app/controllers/api/auth/episodes_controller.rb

```diff
@@ -25,6 +25,10 @@ def visible?
     visible
   end
 
+  def sorted(res)
+    res.order('COALESCE(published_at, released_at) DESC NULLS LAST, id DESC')
```

Comment:
there was no explicit sort on this before? wow, well, this is great.

Reply:
oh, I see, b/c it's the API, not the actual rss feed - makes sense. This looks good!